cmd/convert.go

```diff
@@ -79,6 +79,9 @@ var (
 var convertCmd = &cobra.Command{
 	Use:   "convert",
 	Short: "Convert a Docker Compose file",
+	Example: `  kompose --file docker-voting.yml convert
```

Comment:
We've removed the voting and guestbook examples and we're trying to go away from using `docker-compose.yml / .yml` since compose is an open spec.

I think we should just put `compose.yaml` as even saying `.yml` is deprecated.

If that's good with you 👍

Reply:
No worries, I will update it, so it will be consistent with the #1803.

Reply:
Updated! :fireworks: